WordPress/Classes/ViewRelated/Reader/WPStyleGuide+Reader.swift

```diff
@@ -281,6 +281,46 @@ extension WPStyleGuide {
         button.setTitleColor(disabledColor, for: .disabled)
     }
 
+    @objc public class func applyReaderFollowConversationButtonStyle(_ button: UIButton) {
+        // General
+        button.naturalContentHorizontalAlignment = .leading
+        button.backgroundColor = .clear
+        button.titleLabel?.font = fontForTextStyle(.footnote)
+
+        // Color(s)
+        let normalColor = UIColor.primary(.shade40)
```

Comment:
Based on Chris' designs I think the right color here is just `.primary`. In light mode, the contrast is a little bit low when using `shade40`.

Reply:
Fixed! 7a7d9ce